src/server/index.ts

```diff
@@ -426,6 +426,17 @@ export function msfsNavdataRouter(provider: NavigraphProvider, development: bool
         }
     });
 
+    router.get('/airport/:ident/gates', (req, res) => {
+        try {
+            const ident = parseAirportIdent(req.params.ident);
+            provider.getGates(ident).then((gates) => {
+                res.json(gates);
+            }).catch((error) => errorResponse(error, res));
+        } catch (error) {
+            errorResponse(error, res);
+        }
```

Comment:
do we need the ```.catch((error) => errorResponse(error, res));``` when its all wrapped in a try {} catch {} anyway?

Reply:
Actually... if we awaited the getGates promise then no we wouldn't need it. I copy-pasted without thinking much about it.